docs/quickstart.rst

```diff
@@ -0,0 +1,63 @@
+Quickstart
+==========
+
+Installation
+------------
+
+#. Clone the git repo, and change to the repo directory:
+
+    ::
+
+        git clone git@github.com:release-engineering/releng-sop.git
+        cd releng-sop
+
+
+#. Install the **releng-sop** scripts:
+
+    ::
+
+        python<version> setup.py install
+
+
+Configure environments
+----------------------
+
+The scripts will search the following locations for environment configuration
+(in the order specified below):
+
+#. ``$XDG_CONFIG_HOME/releng-sop/environments/``
+#. ``/etc/releng-sop/environments/``
+
+If ``$XDG_CONFIG_HOME`` is not set or empty, a default equal to ``$HOME/.config``
+is used.
+
+The name of the default environment used be the scripts is ``default``.
+This can be changed using the optional argument ``--env``.
+
+The scripts will look for ``<environment_name>.json`` in the locations mentioned
+above to read environment configuration data.
+
+An example for environment configuration can be found in
+``tests/environments/test-env.json``.
+
+.. literalinclude:: ../tests/environments/test-env.json
+
+
+Setup release data
+------------------
+
+The scripts will search the following locations for release data (in the order
+specified below):
+
+#. ``$XDG_CONFIG_HOME/releng-sop/releases/``
+#. ``/etc/releng-sop/releases/``
+
+If ``$XDG_CONFIG_HOME`` is not set or empty, a default equal to ``$HOME/.config``
+is used.
+
+Release data for a release having ``RELEASE-ID=fedora-24``, will be read from
```

Comment:
I believe all scripts use `RELEASE_ID` with underscore. Should this be consistent?